src/clear-all.js

```diff
@@ -0,0 +1,6 @@
+export default function clearAll() {
+  const tasks = JSON.parse(localStorage.getItem('task.list')) || [];
+  const leaveTasks = tasks.filter((task) => task.completed === false);
+  localStorage.setItem('task.list', JSON.stringify(leaveTasks));
+  window.location.reload();
+}
```

Comment:
- [ ] I would recommend using ES6 syntax. instead of the `function` keyword use the arrow function to make code a little cleaner.